libr/anal/p/anal_avr.c

```diff
@@ -635,7 +647,10 @@ static int avr_op(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *buf, int len)
 			// remove trailing coma (COMETE LA COMA)
 			t = r_strbuf_get (&op->esil);
 			if (t && strlen (t) > 0) {
-				t[strlen(t) - 1] = '\0';
+				t += strlen (t) - 1;
```

Comment:
if strlen( returns 0 it will crash. so pls check that in a separate variable const len = strlen(t); if (len >0) {...